.gitattributes

```diff
@@ -0,0 +1,5 @@
+.* export-ignore
+*.md export-ignore
+tests/* export-ignore
+docs/* export-ignore
+composer.lock export-ignore
```

Comment:
could be removed and should not be commited

Reply:
I know but this will be ignored on release. i did this because travis is always failing. and i cant fix it.

Reply:
i tried it with a `travis_wait 50` [see docs](https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received)

and also i tried with `travis_retry` nothing worked. also on my local `composer install` ran 17hours with no output. => the solution was to require everything new. and commit the composer lock with for travis ci. but it will be ignored on release.

Reply:
@OskarStark there we go! 👍